include/deal.II/matrix_free/portable_matrix_free.h

```diff
@@ -269,6 +271,13 @@ namespace Portable
        * gone through, otherwise the cells in the given local smoothing level.
        */
       unsigned int mg_level;
+
+      /**
+       * The team size used by the underlying kokkos team policy. If -1 is
+       * provided then Kokkos::AUTO will be used, which at the time of writing
+       * is generally suboptimal.
```

Comment:
```suggestion
       * The team size used by the underlying kokkos team policy. If -1 is
       * provided then Kokkos::AUTO will be used.
```
If it's always suboptimal then it shouldn't be optiional.

Reply:
> Actually, we have no evidence that it is suboptimal at this point. Right @RyanMoulday ?

Not for any particular problem we have no. Just what has been said from Kokkos developers. Its possible that in our applications its not too bad. Indeed step-64 seems to pick a reasonable value.